config/locales/es.yml

```diff
@@ -37,6 +37,8 @@ es:
     authorization_handlers:
       census:
         name: Censo
+        fields:
+          born_at: Nacida el
```

Comment:
Maybe "Fecha de nacimiento" would be better?

Reply:
Actually, this is an error. I ended up changing the logic and the final field is called "age" and the message will be "El valor de edad (17) es inválido". I'll update it!

Reply:
I'm going to configure `i18n-tasks` to prevent this, specially since we care mainly about Spanish but the tests run in English.